sos/sosreport.py

```diff
@@ -541,6 +543,28 @@ def compression_type(self, value):
         self._check_options_initialized()
         self._compression_type = value
 
+    @property
+    def encrypt_pass(self):
+        if self._options is not None:
+            return self._options.encrypt_pass
+        return self._encrypt_pass
+
+    @encrypt_pass.setter
+    def encrypt_pass(self, value):
+        self._check_options_initialized()
+        self._encrypt_pass = value
+
+    @property
+    def encrypt_key(self):
+        if self._options is not None:
+            return self._options.encrypt_key
+        return self._encrypt_key
+
+    @encrypt_key.setter
+    def encrypt_key(self, value):
+        self._check_options_initialized()
+        self._encrypt_key = value
```

Comment:
Sorry you had to write all this... It will go away with the presets patches :)

Reply:
Yay! \o/